deployment/helm/node-feature-discovery/templates/clusterrole.yaml

```
@@ -66,3 +66,34 @@ rules:
   - get
   - update
 {{- end }}
+
+---
+{{- if and (eq .Values.topologyGC.enable true) (eq .Values.topologyGC.rbac.create true) ( eq .Values.topologyUpdater.enable true ) }}
```

nit: could be simplified

```suggestion
{{- if and .Values.topologyGC.enable .Values.topologyGC.rbac.create .Values.topologyUpdater.enable }}
```

---

done